src/hevm/src/EVM.hs

```diff
@@ -1339,7 +1368,6 @@ precompiledContract this xGas precompileAddr recipient xValue inOffset inSize ou
           transfer self recipient xValue
           touchAccount self
           touchAccount recipient
-          touchAccount precompileAddr
```

Comment:
why?

Reply:
DELEGATECALL and CALLCODE doesn't touch the address they fetch the code from

Reply:
I don't know a better explanation to "why" than "geth does this"